app/views/decks/_deck_summary.html.erb

```diff
@@ -0,0 +1,106 @@
+<div class="summary">
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="well">
+        <table class="table table-condensed table-striped">
+          <tbody>
+
+            <% if resource == @decks  %>
+
+              <tr>
+                <th><span class="glyphicon glyphicon-book" aria-hidden="true"></span>Decks</th>
+                <td><%= resource.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-published"><%= resource.where(status: 'published').count %> Published</span></li>
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-unpublished"><%= resource.where(status: 'unpublished').count %> Unpublished</span></li>
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-paid"><%= resource.where(tier: 'paid').count %> Paid</span></li>
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-free"><%= resource.where(tier: 'free').count %> Free</span></li>
+                    </a>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-bookmark" aria-hidden="true"></span>Categories</th>
+                <td><%= @categories.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <% @categories.each do |category| %>
+                      <li class="list-item-label">
+                        <span class="label label-default"><%= @category_deck_links.where(category_id: category.id).count %> <%= category.name %></span>
```

Comment:
instead of counting the associations within the `category_deck_links` table, we can count the number of `decks` per category (this is essentially what the `category_deck_links` table allows us to do, also referred to as a `has_many through` association).
```ruby
Category.find_each do |category|
  category.decks.count
  category.name
end
```